java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -243,4 +243,30 @@ public interface StringBaseCommands {
      * }</pre>
      */
     CompletableFuture<String> getrange(String key, int start, int end);
+
+    /**
+     * Appends a value to a key. If `key` does not exist it is created and set as an empty string, so
+     * `APPEND` will be similar to `SET` in this special case.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/append/">redis.io</a> for details.
+     * @param key The <code>key</code> to which the value will be appended.
+     * @param value The value to append
+     * @return Response from Redis. The length of the string after appending the value.
+     * @example
+     *     <pre>{@code
+     * String value = client.append("key", "value").get();
+     * assert value.equals(5);
+     * Indicates that "Hello" has been appended to the value of <code>key</code>, which was initially empty,
+     * resulting in a new value of "Hello" with a length of 5 - similar to the set operation.
+     *
+     * String value = client.append("key", " world").get();
```

Comment:
In java we ~usually~ always add only one example

Reply:
@Yury-Fridlyand i copied over the python code. is it better if I just remove one example?

Reply:
Yes. All python client examples (where they are) are more verbose.